02/app.js

```diff
@@ -1 +1,9 @@
-console.log('DOM');
\ No newline at end of file
+const dataUrl = document.querySelectorAll('[data-url]');
+
+if(dataUrl) {
```

Comment:
Jeśli mamy `querySelectorAll` to nie musimy sprawdzać czy nie jest `null` bo brak elementów to pusta tablica i błędu nie będzie ;)